packages/remote-config/android/src/main/java/io/invertase/firebase/config/UniversalFirebaseConfigModule.java

```diff
@@ -143,9 +155,53 @@ Task<FirebaseRemoteConfigInfo> ensureInitialized(String appName) {
     return ensureInitializedTask;
   }
 
+  ConfigUpdateListenerRegistration onConfigUpdated(String appName, Callback callback) {
+    FirebaseApp firebaseApp = FirebaseApp.getInstance(appName);
+
+    return FirebaseRemoteConfig.getInstance(firebaseApp).addOnConfigUpdateListener(new ConfigUpdateListener() {
+      @Override
+      public void onUpdate(ConfigUpdate configUpdate) {
+        Set<String> updatedKeys = configUpdate.getUpdatedKeys();
+        List<String> updatedKeysList = new ArrayList<>(updatedKeys);
+
+        Map<String, Object> results = new HashMap<>();
+        results.put("updatedKeys", updatedKeysList);
+        callback.invoke(Arguments.makeNativeMap(results), null);
+      }
+
+      @Override
+      public void onError(FirebaseRemoteConfigException error) {
+        WritableMap userInfoMap = Arguments.createMap();
+
+        if (error == null) {
```

Comment:
I don't think this can be `null`. Remove.

Reply:
Done